docs/start_guide.md

```diff
@@ -0,0 +1,279 @@
+      marathon:
+        - file://artifacts/marathon/helloapache.json
+```
+
+__Graph__ is the most important section. In here we will indicate all the default parameters as well as all associated artifacts.
+
+```yaml
+params:
+  - name: image
+    description: The webserver image
+    default: centos/httpd
+```
+There will likely be many parameters that need to be exposed at deployment. It's best to provide defaults whenever possible. Variable templating is used within artifact files. For example: `$image` within `artifacts/kubernetes/hello-apache-pod.json` becomes `centos/httpd`.
+
+**NOTE:** Not providing a default variable will require Atomic App to ask the user. Alternatively, an `answers.conf` file can be provided.
+
+```yaml
+artifacts:
+  docker:
+    - file://artifacts/docker/hello-apache-pod_run
+  kubernetes:
+    - file://artifacts/kubernetes/hello-apache-pod.json
+  marathon:
+    - file://artifacts/marathon/helloapache.json
+```
+In order to use a particular provider, name as well as a file location required. Each file is a variable-subtituted template of how your Atomic App container is ran. We go more into detail below.
+
+```yaml
+kubernetes:
+  - file://artifacts/kubernetes/hello-apache-pod.json
+  - file://artifacts/kubernetes/hello-apache-service.json
+  - file://artifacts/kubernetes/hello-apache-rc.json
+```
+Multiple files may also be specified. For example, specifying a pod, service and replication controller for the `kubernetes` provider.
+
+### ~/helloapache/artifacts/docker/hello-apache-pod_run
+```sh
+docker run -d -p $hostport:80 $image
+```
+Each artifact uses variable replacement values. For our Docker provider, we substitute the port number with `$hostport` as indicated by our graph in our `Nulecule` file. The same as our `$image` variable.
+
+### ~/helloapache/artifacts/kubernetes/hello-apache-pod.json
+```json
+"image": "$image",
+"name": "helloapache",
+"ports": [
+    {
+        "containerPort": 80,
+        "hostPort": $hostport,
+        "protocol": "TCP"
+    }
+```
+
+Similarly, the kubernetes provider uses both `$image` and `$hostport` variables for pod deployment.
+
+### ~/helloapache/answers.conf.sample
+
+`answers.conf.sample` is an answers file generated while fetching. It is a generated ini file that provides parameter answers for non-interactive deployments.
+
+```ini
+[helloapache-app]
+image = centos/httpd
+hostport = 80
+
+[general]
+namespace = default
+provider = kubernetes
+```
+
+Default values such as the provider as well as the namespace can be provided.
+
+In order to use an answers file, simply specify the location of the file when deploying:
+```sh
+cp answers.conf.sample answers.conf
+sudo atomicapp run -a answers.conf .
+```
+
+### Conclusion
+
+Now you know how to build your very own first app! After you have created the necessary files go ahead and build/run it!
+
+```sh
+sudo docker build -t myapp .
+sudo atomicapp run myapp
+```
```

Comment:
we may want to emphasize here that you can push the image to a registry and deploy it elsewhere... so adding a push step in between these two could be useful.